dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/export/enrollment/HibernateEnrollmentStore.java

```diff
@@ -306,10 +305,4 @@ protected Enrollment postProcessObject(Enrollment enrollment) {
   public Set<String> getOrderableFields() {
     return ORDERABLE_FIELDS;
   }
-
```

Comment:
Is there any other place where we are setting the status to `CANCELLED` before we soft delete the enrollment.
If no, we are changing the behavior here.

Reply:
you're right, the only thing is the method needs to be called `delete` instead of `deleteEnrollment`